### PR TITLE
[#176615471] Test suite POST wallet

### PR DESCRIPTION
**Pull request by GiovanniMancini**
This PR proposes a suite of unit tests covering `addWalletUsingPOST` method of PM client.

#### List of Changes

- Unit test suite for `addWalletUsingPOST`
- Endpoint `wallet` to PM Stub

#### Motivation and Context

This PR is required to assert that the PM shows the expected behaviour when POST `wallet` is called via autogenerated client

#### How Has This Been Tested?

Suite of unit tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by infantesimone — APPROVED** [review]
Well done 💪

File: src/__integrations__/pm.ts [hunk 1]
```diff
@@ -29,6 +31,79 @@ walletRouter.post('/pp-restapi/v3/users/actions/start-session', function (req, r
...
+              expireYear: decodedReq.data?.creditCard?.expireYear,
+              brandLogo: 'http://localhost:8080/wallet/assets/img/creditcard/generic.png',
+              flag3dsVerified: false,
+              brand: 'OTHER',
+              hpan: myFake.random.alphaNumeric(64),
+              onUs: false,
+            },
+            psp: {
+              id: 8,
+              idPsp: 'POSTE1',
```
Comment (pasqualespica) [inline src/__integrations__/pm.ts:74]: Could be useful to put all hardcoded data in to `mock` module file ?

File: src/__integrations__/pmClient.addWallet.integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+          },
+          language: 'it',
+        })
+      ).fold(
+        err => (err.pop()?.value as Response).status,
+        () => undefined,
+      ),
+    ).toEqual(422);
+  });
+});
```
Comment (pasqualespica) [inline src/__integrations__/pmClient.addWallet.integration.test.ts:190]: Does't miss a test on answer response fields ?

**Review by pasqualespica — APPROVED** [review]
Great work 👍 
it would be to understand whether or not we need a file with mock data

File: src/__integrations__/pmClient.addWallet.integration.test.ts (hunk 2 above)
Comment (GiovanniMancini) [reply src/__integrations__/pmClient.addWallet.integration.test.ts:190]: @pasqualespica I agree. I'll add it

File: src/__integrations__/pm.ts (hunk 1 above)
Comment (GiovanniMancini) [reply src/__integrations__/pm.ts:74]: @pasqualespica I totally agree, but in my opinion we should avoid the standard  `__mock__` folder used by jest

**Pull request merged**